src/odemis/driver/hamamatsurx.py

```diff
@@ -807,15 +807,27 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
         parent.DevParamSet(self.location, "Delay A", 0)
         parent.DevParamSet(self.location, "Delay B", 0.00000002)
         parent.DevParamSet(self.location, "Burst Mode", "Off")
+        parent.DevParamSet(self.location, "PLL Mode", "Locked")  # Set phase locked loop mode, only used in synchroscan
 
         self._metadata[model.MD_TRIGGER_DELAY] = self.GetTriggerDelay()
         # Note: trigger rate (repetition rate) corresponds to the ebeam blanking frequency (read only in RemoteEx)
         self._metadata[model.MD_TRIGGER_RATE] = int(self.parent.DevParamGet(self.location, "Repetition Rate")[0])
 
+        # VAs
+        pllMode = self.GetPLLMode()
+        self.phaseLocked = model.BooleanVA(pllMode, setter=self._setPLLMode)  # default False see set params above
+
         triggerDelay = self.GetTriggerDelay()
         range_trigDelay = self._getTriggerDelayRange()
         self.triggerDelay = model.FloatContinuous(triggerDelay, range_trigDelay, setter=self._setTriggerDelay, unit="s")
 
+        # triggerDelay is Delay A, that's why delays start with Delay B
+        delays= ["Delay B", "Delay C", "Delay D", "Delay E", "Delay F", "Delay G", "Delay H"]
+        for d in delays:
+            delay = self.GetDelayByName(d)
+            range_delay = self._getDelayRangeByName(d)
+            self.delay = model.FloatContinuous(delay, range_delay, setter=self._setDelayByName, unit="s")
```

Comment:
You're overriding the VA for every parameter! There should be one VA per parameter, named `delayB`, `delayC`..., `delayH`. => use `setattr()`.
It should also only happen on the component which have such parameters. (So, no on the synchroscan delay unit)